src/main/scala/io/iohk/ethereum/crypto/ConcatKDFBytesGenerator.scala

```diff
@@ -16,7 +16,7 @@ class ConcatKDFBytesGenerator(digest: Digest) {
     * @param outputLength length of output that will be produced by this method,
     *                     maximum value is (digest output size in bits) * (2^32 - 1) but it should not be a problem
     *                     because we are using Int
-    * @throws IllegalArgumentException ("Output length too large") when outputLength is greater than (digest output size in bits) * (2^32 - 1)
+    * @throws scala.IllegalArgumentException ("Output length too large") when outputLength is greater than (digest output size in bits) * (2^32 - 1)
```

Comment:
Yes, there were some warnings on the generation of the zip file related to the Scala Docs, without those I think there were some problems with the linking between them. So the changes on the comments were all related to that.